zboss/doc/zboss_configuration.rst

```diff
@@ -5,9 +5,26 @@ ZBOSS configuration
 
 The ZBOSS stack is distributed as a set of precompiled linkable libraries.
```

Comment:
I would introduce the NCP and certification library variants here briefly, maybe with a link to nrfxlib section that describes their differences in details (what is certified and what is experimental and info similar to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/openthread/README.html).

Then, I would continue with two subsections, one for each library variant, with information about different libraries for Zigbee device roles and the experimental status of the NCP one.

We would need more info about the certification library and how to configure its use ("enabled by default" is also a valuable piece of information), as well as where the files for different variants can be found.

Reply:
* Note about the default choice of libraries added.
* Configuration page split into two sections.